rero/rero-ils-ui
# document: adapts detailed view for series statements fields

* Adds new fields: issuance, series statement, partOf.
* Removes partOf field from brief view.
* Corrects notes display.
* Updates ngx-formly to v5.9.1.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

https://tree.taiga.io/project/rero21-reroils/us/1432?milestone=266303

## Dependencies

My PR depends on `rero-ils#1028`'s PR(s):

* rero/rero-ils#1028 & https://github.com/rero/ng-core/pull/193

## How to test?
Go to a detail view of a document with issuance, series statement and part of fields ( create one if needed).
Check that the fields are correctly displayed.
Check the editor too.

## Code review check list
